src/core/unit_tests/Ellipsoid_test.cpp

```diff
@@ -35,7 +35,7 @@ bool check_distance_function(const Shapes::Shape &s) {
   for (int i = 0; i < N; i++) {
     for (int j = 0; j < N; j++) {
       double theta = 2. * i / N * M_PI;
-      double v = j / (N - 1);
+      double v = j / (N - 1.);
```

Comment:
This changes the behavior, but the old one was obviously wrong (`v` would always be zero).